lib/components/molecules/Form/FieldFileInput/FieldFileInput.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Input from '../../../atoms/Input';
+import Button from '../../../atoms/Button';
+import styles from './FieldFileInput.style';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  label: Node,
+  labelProps?: Object,
+  fieldProps?: Object,
+  value?: string | number,
+  placeholder?: string,
+  onChange?: Function,
+  onBlur?: Function,
+};
+
+const FieldFileInput = ({
+  id,
+  className,
+  type,
+  name,
+  label,
+  onChange,
+  onBlur,
+  labelProps,
+  fieldProps,
+  ...others
+}: Props): Node => (
+  // TODO: Add support for label position
+  <div className={className}>
+    <Field
+      id={id}
+      name={name}
+      type={type}
+      render={({ field, form }) => {
+        const { handleChange, handleBlur } = form;
+        return (
+          <Input
```

Comment:
Think of it this way.
Where is the CSS for input really residing ? 
FieldInput is just a wrapper and should have CSS only for label  and outer styling.

You have 2 options:
Write CSS in Input.style.js for [type=file]

or better: create a new component named FileInput, Use <Input type="file"> inside it and write css in FileInput.style.js

And then wrap everything in this component

Reply:
I followed second approach as suggested. Also, it was giving me problem in one more case i.e. value field as well, when i tried uploaded file and it was not able to set explicit value prop.